scripts/backup

```diff
@@ -37,10 +37,10 @@ domain=$(ynh_app_setting_get --app=$app --key=domain)
 ynh_print_info --message="Declaring files to be backed up..."
 
 #=================================================
-# BACKUP THE APP MAIN DIR
+# BACKUP THE APP DATA DIRS
 #=================================================
 
-ynh_backup --src_path="$final_path" --is_big
+ynh_backup --src_path="$final_path/data" --is_big
```

Comment:
The backup script should backup all the file not just `/data` and the restore script should restore the app from backup not reinstall the app with `npm` like install script... Am I missing something?

Reply:
The problem is the node_modules and .Cache repo weight 2.6Go total so I thought I'd exclude them from backup since there's still a lock file that is preserved and allows to redownload all this if needed. 
I wanted an `--exclude` flag, basically.

Feel free to close if it's an antipattern.